### ISSUE TITLE
No notifications to own phone anymore (*note to self* and all other groups) when using dbus 

### ISSUE THREAD

**Issue by atticus-sullivan** [AsamK/signal-cli#1512]
Hi,
I really don't want to pressure you, I just want to avoid that the follow up of https://github.com/AsamK/signal-cli/issues/1087 gets lost. With the commit fixing that issue, the result is when using dbus, one never gets a notification in any case anymore. Maybe you forgot to add the respective parameter to the dbus api :shrug: .

The sad thing about this is that the idea of using singal_cli to send notifications from a server does not work anymore. Before https://github.com/AsamK/signal-cli/commit/3290a5bf4d642beaef6584a700aab8f2da3af6e0 at least it was possible to send these notifications via the *note to self* chat and get notifications. Now the whole feature of notifications is broken.

Also see https://github.com/AsamK/signal-cli/issues/1087#issuecomment-2011018969 about this issue.

**Comment by atticus-sullivan** [comment]
Personally, this not working was the reason to switch to the jsonRPC interface. Thus, this issue is closed / not really relevant for me anymore

**Comment by javl** [comment]
@atticus-sullivan Are you able to receive notifications when using the jsonRpc interface? Running the command below works to send a message using jsonRpc, but it still does not trigger any notification (on IOS)

```echo '{"jsonrpc":"2.0","method":"send","params":{"recipient":"+xxxxxxxxxxx","message":"MESSAGE"}, "id": 2}' | ./signal-cli -u +xxxxxxxxxxx jsonRpc```

I'd love to be able to send myself notifications from scripts, but don't really want to get a second SIM card just for that.

**Comment by atticus-sullivan** [comment]
Yes with the jsonRpc interface notifications work just fine for me. But I also have no IOS device so I'm afraid that's of no help for you.

Nonetheless: Are you sure notifications from Signal generally work? (I heard sometimes there are issues with the delivery of push notifications, especially when the app wasn't opened in a while)
